services/brig/test/integration/API/User/Account.hs

```diff
@@ -299,14 +299,19 @@ testCreateUserBlacklist brig aws =
                                                , "password" .= defPassword
                                                ]
 
-    publishMessage :: Text -> Email -> AWS.Env -> Http ()
-    publishMessage typ em env = do
+    -- If there is no queue available, we need to force it either by publishing an event or using the API
+    forceBlacklist :: Text -> Email -> Http ()
+    forceBlacklist typ em = case aws^.AWS.sesQueue of
+        Just queue -> publishMessage typ em queue
+        Nothing    -> Bilge.post (brig . path "i/users/blacklist" . queryItem "email" (toByteString' em)) !!! const 200 === statusCode
```

Comment:
While this is fine in the tests; how will backlisting work with an SMTP server configured? (This is a little cheating: the tests testing blacklisting will pass even when SMTP instead of SES is configured; however no blacklisting is ever done when SMTP is used in production)